### DISCUSSION TITLE
Rounded image

### DISCUSSION THREAD

**Discussion by MuttakinHasib** [zpao/qrcode.react#219] [Q&A]
I want to make the center image a rounded shape ... how can I do this?

**Comment by MuttakinHasib** [comment]
@zpao hey, please help me

**Comment by zpao** [comment]
I don't have any support for masking the image, so you're going to be stuck with rectangles. What you can do with that though is use a transparent PNG with the area you want rounded already cropped.

**Reply by zpao** [reply]
If you have a proposal for how you might want this to work otherwise, please submit a new issue.
